conf.py

```diff
@@ -34,6 +34,7 @@
 extensions = [
         'myst_parser', # there's also myst_nb, which supports embedding Jupyter notebooks, but is heavier.
         'sphinx_panels',
+        'sphinx_design',
```

Comment:
This is a strange dependency. It pulls in bootstrap? But bootstrap has already been deprecated for several years by the existence of https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids. If I look at the web debugger in https://sphinx-design.readthedocs.io/en/sbt-theme/grids.html#grid-options I see they're not using that at all though:

![2021-12-04-173551_337x69_scrot](https://user-images.githubusercontent.com/987487/144726613-ff79db2f-90c1-4269-a745-ba95b7907970.png)

Reply:
this is needed, otherwise on compilation i get:
~~~
/Users/julien/code/neuro.polymtl.ca/gallery.txt:2: WARNING: Unknown directive type "grid".
~~~